README.md

```diff
@@ -89,8 +89,8 @@ set to `true` if you don't want to see the launcher.
 | `params`               | See below          | Essentially used to customize the image size.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
 | `storage`              | `"local"`          | Specifies the storage location of the conversation state in the browser. `"session"` defines the state to be stored in the session storage. The session storage persists on reload of the page, and is cleared after the browser or tab is closed, or when `sessionStorage.clear()`is called. `"local"` defines the state to be stored in the local stoage. The local storage persists after the the browser is closed, and is cleared when the cookies of the browser are cleared, or when `localStorage.clear()`is called. |
 | `customComponent`      | `null`             | Custom component to be used with custom responses. E.g.: `customComponent={ (messageData) => (<div>Custom React component</div>)` }                                                                                                                                                                                                                                                                                                                                                                                          |
-
-| `onWidgetEvent`        | `{}`             | call custom code on a specific widget event ( onChatOpen, onChatClose, onChatHidden, are available for now ), add a function to the desired object property in the props to have it react to the event.
+| `onWidgetEvent`        | `{}`             | call custom code on a specific widget event ( onChatOpen, onChatClose, onChatHidden, are available for now ), add a function to the desired object property in the props to have it react to the event. |
+| `marginBottom`        | See below             | Specifies a custom value for margin-bottom CSS property for `.conversation-container` class .
```

Comment:
I think this is better with something like "...for the conversational container (above the close button).", it's is more readable for users.

Reply:
How about this:
Specifies a custom value for margin-bottom CSS property for `.conversation-container` class, this changes the height that the container will appear above the close button.

Reply:
Nice. I would just change from `"the height that"` to `"how much"`... What do you think? For me `"height"` is a strong word in CSS context, may causing confusion with the `height` property haha.

Reply:
You're correct, i'll change that.